source/kubernetes/direct-csi.rst

```diff
@@ -0,0 +1,135 @@
+==============================================
+MinIO Direct Container Storage Interface (CSI)
+==============================================
+
+.. default-domain:: minio
+
+.. contents:: On This Page
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+The MinIO Direct Container Storage Interface (CSI) driver for Kubernetes
+supports creating Persistant Volumes (``PV``) from disks attached to a 
+:kube-docs:`Node <concepts/architecture/nodes/>`. The
+:doc:`MinIO Kubernetes Operator </kubernetes/operator-reference>` and
+:doc:`MinIO Kubernetes Plugin </kubernetes/kubectl-plugin-reference>`
+support using MinIO Direct CSI volumes when creating or expanding
+MinIO deployments. 
+
+Complete documentation on the :kube-docs:`Container Storage Interface
+<concepts/storage/volumes/#csi>` is out of scope for this documentation. 
+The MinIO documentation focuses on guidance and reference material related
+to using the MinIO Direct CSI plugin. The documentation makes a best effort
+to summarize or link to Kubernetes-specific concepts and resources, and is
+neither intended nor designed to replace Kubernetes-centric documentation.
+
+.. TODO: Compare/Contrast Direct CSI vs ``local`` and other volume types.
+
+Quickstart
+----------
+
+The following command configures and deploys the MinIO Direct CSI plugin on a
+Kubernetes cluster.
+
+.. code-block:: shell
+   :class: copyable
+
```

Comment:
Add `# Set the environment variables that will get picked up by kustomized based install below`

Reply:
This is to spoon-feed what is really going on.

Reply:
Done